### PR TITLE
Update es.yml

### PR DESCRIPTION
**Pull request by 103percent**
Some more spanish strings

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -131,21 +131,21 @@ es:
...
       reminder: "So please remember what these workshops are for – to open up tech to people that are underrepresented in the industry."
     teacher_guide:
-      laptop: "You don't need a laptop for our workshops. You are there to watch and guide the students. If you don't feel comfortable with the tutorial you should spend time on it before coming if you want to help out."
-      keyboard: "Do not take over the keyboard! This can be off-putting and scary."
-      typing: "Anime a los estudiantes a escribir y no a copiar y pegar."
-      questions: "Explain that there are no bad questions."
+      laptop:  "No necesita una computadora portátil para nuestros talleres. Usted está ahí para ver y guiar a los estudiantes. Si no se siente cómodo con el tutorial, debe pasar tiempo antes de venir si quiere ayudarlo."
+      keyboard:  "¡No tomes el teclado! Esto puede estar fuera de lugar y aterrador."
+      typing:  "Explique que no hay malas preguntas."
+      questions:  "Explique a los estudiantes que está bien cometer errores."
```
Comment (matyikriszta) [inline config/locales/es.yml:137]: @103percent this line should be the translation for `Explain that there are no bad questions.` and the next line, `mistakes` is `Explain to students that it's OK to make mistakes.` aka `Explique a los estudiantes que está bien cometer errores.`

Comment (103percent) [reply config/locales/es.yml:137]: Thanks for the save here!

File: config/locales/es.yml [hunk 2]
```diff
@@ -131,21 +131,21 @@ es:
       abuse: "We’re a fairly relaxed bunch and we trust you, so we don’t police students as to their eligibility. However if it ever looks like this is being abused we may have to change that – which we really don’t want to do."
       reminder: "So please remember what these workshops are for – to open up tech to people that are underrepresented in the industry."
     teacher_guide:
-      laptop: "You don't need a laptop for our workshops. You are there to watch and guide the students. If you don't feel comfortable with the tutorial you should spend time on it before coming if you want to help out."
-      keyboard: "Do not take over the keyboard! This can be off-putting and scary."
-      typing: "Anime a los estudiantes a escribir y no a copiar y pegar."
-      questions: "Explain that there are no bad questions."
+      laptop:  "No necesita una computadora portátil para nuestros talleres. Usted está ahí para ver y guiar a los estudiantes. Si no se siente cómodo con el tutorial, debe pasar tiempo antes de venir si quiere ayudarlo."
+      keyboard:  "¡No tomes el teclado! Esto puede estar fuera de lugar y aterrador."
+      typing:  "Explique que no hay malas preguntas."
```
Comment (matyikriszta) [inline config/locales/es.yml:136]:
```suggestion
      typing: "Anime a los estudiantes a escribir y no a copiar y pegar."
```

File: config/locales/es.yml (hunk 1 above)
Comment (matyikriszta) [inline config/locales/es.yml:137]:
```suggestion
      questions:  "Explique que no hay malas preguntas."
```

File: config/locales/es.yml [hunk 3]
```diff
@@ -131,21 +131,21 @@ es:
...
+      questions:  "Explique a los estudiantes que está bien cometer errores."
       mistakes: "Explain to students that it's OK to make mistakes."
```
Comment (matyikriszta) [inline config/locales/es.yml:138]:
```suggestion
      mistakes: "Explique a los estudiantes que está bien cometer errores."
```

**Review by matyikriszta — APPROVED** [review]

**Pull request merged**